samples/graph-activity-feed/csharp/ActivityFeedNotification/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
+  "clientAppId": "",
+  "tenantId": "",
+  "teamId": "",
+  "channelId": "",
+  "UserName": "",
```

Comment:
User name and password should not be in config file. Let's revisit the scenario.

Reply:
Removed from appSetting.json, We have used Username/password auth for Graph API chat message call